src/bin/ion/commands/beta/symtab/filter.rs

```diff
@@ -32,44 +32,30 @@ impl IonCliCommand for SymtabFilterCommand {
     }
 
     fn run(&self, _command_path: &mut Vec<String>, args: &ArgMatches) -> Result<()> {
-        let mut output: Box<dyn Write> = if let Some(output_file) = args.get_one::<String>("output")
-        {
-            let file = File::create(output_file).with_context(|| {
-                format!(
-                    "could not open file output file '{}' for writing",
-                    output_file
-                )
-            })?;
-            Box::new(BufWriter::new(file))
-        } else {
-            Box::new(stdout().lock())
-        };
-
         let lift_requested = args.get_flag("lift");
 
-        if let Some(input_file_names) = args.get_many::<String>("input") {
-            // Input files were specified, run the converter on each of them in turn
-            for input_file in input_file_names {
-                let file = File::open(input_file.as_str())
-                    .with_context(|| format!("Could not open file '{}'", &input_file))?;
+        args.for_each_input(|output, input_file_name| {
+            if input_file_name == "--" {
```

Comment:
`-` stands in for stdin, `--` is the separator between options and arguments that look like options, e.g. when you want to operate on a file named `-f` or some case like that. See guidelines 10 and 13 in this section of the [POSIX standard](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html).

> **Guideline 10:**
>    The first -- argument that is not an option-argument should be accepted as a delimiter indicating the end of options. Any following arguments should be treated as operands, even if they begin with the '-' character.
> [...]
> **Guideline 13:**
>     For utilities that use operands to represent files to be opened for either reading or writing, the '-' operand should be used to mean only standard input (or standard output when it is clear from context that an output file is being specified) or a file named -.

```suggestion
            if input_file_name == "-" {
```